npc/mapflag/zone.txt

```diff
@@ -131,3 +131,19 @@ umbala	mapflag	zone	Towns
 veins	mapflag	zone	Towns
 xmas	mapflag	zone	Towns
 yuno	mapflag	zone	Towns
+
+// WoE:TE
+te_prt_gld	mapflag	zone	WoE TE
+te_prtcas01	mapflag	zone	WoE TE
+te_prtcas02	mapflag	zone	WoE TE
+te_prtcas03	mapflag	zone	WoE TE
+te_prtcas04	mapflag	zone	WoE TE
+te_prtcas05	mapflag	zone	WoE TE
+te_alde_gld	mapflag	zone	WoE TE
+te_aldecas1	mapflag	zone	WoE TE
+te_aldecas2	mapflag	zone	WoE TE
+te_aldecas3	mapflag	zone	WoE TE
+te_aldecas4	mapflag	zone	WoE TE
+te_aldecas5	mapflag	zone	WoE TE
+teg_dun01	mapflag	zone	WoE TE
+teg_dun02	mapflag	zone	WoE TE
```

Comment:
suffer the rathena script language symptom again ? XD
`[Warning]: npc_parse_mapflag: Invalid zone 'WoE TE'! removing flag from te_prt_gld in file 'npc/zzz.txt', line '1'.`
I believe we use `map_zone_db.conf`
https://github.com/HerculesWS/Hercules/blob/master/db/re/map_zone_db.conf#L116